src/scheduler/pbsfs.c

```diff
@@ -72,10 +72,9 @@ static void print_fairshare(group_info *root, int level);
 #define FS_PRINT 4
 #define FS_PRINT_TREE 8
 #define FS_DECAY 16
-#define FS_ADD 32
-#define FS_COMP 64
-#define FS_TRIM_TREE 128
-#define FS_WRITE_FILE 256
+#define FS_COMP 32
+#define FS_TRIM_TREE 64
+#define FS_WRITE_FILE 128
```

Comment:
Technically, there's nothing wrong with defining these macros with decimal values. Probably better to express powers of two in hex or octal. Didn't see this in my previous review because it hadn't changed.

Reply:
I'd rather leave them alone.  pbsfs is part of the scheduler, and the scheduler uses decimal powers of 2.  If I were to change these, I'd want to change the rest of the scheduler's defines.

Reply:
Makes sense to me.